.github/workflows/run-osp-extended-tests.yml

```diff
@@ -0,0 +1,151 @@
+name: openstack-extended-tests
+on:
+  workflow_dispatch:
+    name:
+      description: 'run e2e openstack  extended tests on Self hosted runner'
+  workflow_call:
+  issue_comment:
+    types: [ created ]
+
+# ensures that only a single workflow  will run at the same time
+# if another workflow triggered it will pending until first completion.
+concurrency:
+  group: hosted-e2e
+
+# A workflow run is made up of one or more jobs that can run sequentially
+jobs:
+  create_ovirt_runner:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/test-osp-extended')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR branch
+        uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+      - name: Set latest commit status as pending
+        if: always()
+        uses: myrotvorets/set-commit-status-action@master
+        with:
+          sha: ${{ steps.comment-branch.outputs.head_sha }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          status: pending
+
+      - name: Checkout forklift
+        uses: actions/checkout@v3
+
+      - id: codeowner
+        uses: SvanBoxel/codeowners-action@v1
+        with:
+          path: './.github/CODEOWNERS'
+
+      - name: fail if the commenter doesnt exists in codeowners
+        run: |
+          [[ $(echo '${{ steps.codeowner.outputs.codeowners }}' | \
+          jq  -r '."tests/" | index("@${{ github.event.comment.user.login }}")') -eq 1 ]]
```

Comment:
the other comment on this check should have been placed here but also applies there to the message by the bot